src/main/scala/viper/gobra/Gobra.scala

```diff
@@ -143,6 +165,13 @@ trait GoVerifier extends StrictLogging {
     if (allErrors.isEmpty) VerifierResult.Success else VerifierResult.Failure(allErrors)
   }
 
+  private def createSubmitter(allowSubmission: Boolean, id: String, verifier: String): Unit = {
+    if (allowSubmission) {
+      val submitter = new ManualProgramSubmitter(true, "", "Gobra", verifier, Array())
```

Comment:
are we actually submitting anything if the second parameter is always `""`?

Reply:
Btw, I find it a bit odd that we have a flag `ProgramSubmitter` has a field `allowSubmission`. Maybe I misunderstand its purpose, but if this object was registered, it should be submitted. Otherwise, we should not register it. I don't see the need for this field.